x/poe/keeper/keeper.go

```diff
@@ -5,6 +5,8 @@ import (
 	"sync"
 	"time"
 
+	types2 "github.com/tendermint/tendermint/abci/types"
```

Comment:
```suggestion
	abcitypes "github.com/tendermint/tendermint/abci/types"
```

And subsequent changes.